Index: ddclient-3.6.7/ddclient
===================================================================
--- ddclient-3.6.7.orig/ddclient	2006-04-05 01:53:59.000000000 +0200
+++ ddclient-3.6.7/ddclient	2006-04-05 01:58:50.000000000 +0200
@@ -1701,7 +1701,7 @@
 	# Protect special HTML characters (like '?')
 	$queryif =~ s/([\?&= ])/sprintf("%%%02x",ord($1))/ge;
 
-	$url   = "http://${arg}/level/1/exec/show/ip/interface/brief/${queryif}/CR";
+	$url   = "http://".opt('fw')."/level/1/exec/show/ip/interface/brief/${queryif}/CR";
 	$reply = geturl('', $url, opt('fw-login'), opt('fw-password')) || '';
 	$arg   = $url;
 
