From: Torsten Landschoff <torsten@debian.org>
Date: Wed, 5 Apr 2006 02:01:22 +0200
Subject: From version 3.6.7-1, config_path.diff

Forwarded: no
---
 ddclient | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ddclient b/ddclient
index 9176862..9180016 100755
--- a/ddclient
+++ b/ddclient
@@ -33,7 +33,7 @@ my $program   = $programd;
 $program  =~ s/d$//;
 my $now       = time;
 my $hostname  = hostname();
-my $etc       = ($program =~ /test/i) ? './'   : '/etc/ddclient/';
+my $etc       = ($program =~ /test/i) ? './'   : '/etc/';
 my $cachedir  = ($program =~ /test/i) ? './'   : '/var/cache/ddclient/';
 my $savedir   = ($program =~ /test/i) ? 'URL/' : '/tmp/';
 my $msgs      = '';
