Index: ddclient-3.7.3/ddclient
===================================================================
--- ddclient-3.7.3.orig/ddclient	2007-08-08 20:55:25.264000630 +0200
+++ ddclient-3.7.3/ddclient	2007-08-08 20:55:25.296995597 +0200
@@ -24,7 +24,7 @@
 $program  =~ s/d$//;
 my $now       = time;
 my $hostname  = hostname();
-my $etc       = ($program =~ /test/i) ? './'   : '/etc/ddclient/';
+my $etc       = ($program =~ /test/i) ? './'   : '/etc/';
 my $cachedir  = ($program =~ /test/i) ? './'   : '/var/cache/ddclient/';
 my $savedir   = ($program =~ /test/i) ? 'URL/' : '/tmp/';
 my $msgs      = '';
