Index: ddclient-3.7.3/ddclient
===================================================================
--- ddclient-3.7.3.orig/ddclient	2007-08-08 20:55:23.753231108 +0200
+++ ddclient-3.7.3/ddclient	2007-08-08 20:56:41.328396530 +0200
@@ -986,6 +986,8 @@
 sub init_config {
     %opt = %saved_opt;
 
+    $opt{'daemon'}=interval($opt{'daemon'}) if defined $opt{'daemon'};
+
     ## 
     $opt{'quiet'}   = 0 if   opt('verbose');
 
