Index: ddclient-3.6.7/ddclient
===================================================================
--- ddclient-3.6.7.orig/ddclient	2006-04-05 01:38:15.000000000 +0200
+++ ddclient-3.6.7/ddclient	2006-04-05 01:39:25.000000000 +0200
@@ -877,6 +877,8 @@
 sub init_config {
     %opt = %saved_opt;
 
+    $opt{'daemon'}=interval($opt{'daemon'}) if defined $opt{'daemon'};
+
     ## 
     $opt{'quiet'}   = 0 if   opt('verbose');
 
