Index: ddclient-3.7.3/sample-etc_ppp_ip-up.local
===================================================================
--- ddclient-3.7.3.orig/sample-etc_ppp_ip-up.local	2007-07-07 14:37:47.000000000 +0200
+++ ddclient-3.7.3/sample-etc_ppp_ip-up.local	2007-08-14 22:56:23.194306140 +0200
@@ -22,6 +22,10 @@
 IP=${IP:-$IPLOCAL}
 IP=${IP:-$4}
 
+IFACE=
+IFACE=${IFACE:-$PPP_IFACE}
+IFACE=${IFACE:-$1}
+
 ## update the DNS server unless the IP address is a private address
 ## that may be used as an internal LAN address (or PPtP tunnel).
 
@@ -34,7 +38,7 @@
 		;;
 *)		(
 			sleep 5
-			ddclient -daemon=0 -syslog -use=if -if=$1 >/dev/null 2>&1
+			ddclient -daemon=0 -syslog -use=if -if=$IFACE >/dev/null 2>&1
 		) &
 		;;
 esac
