Index: ddclient-3.7.3/sample-etc_ppp_ip-up.local
===================================================================
--- ddclient-3.7.3.orig/sample-etc_ppp_ip-up.local	2007-08-08 20:55:23.534264513 +0200
+++ ddclient-3.7.3/sample-etc_ppp_ip-up.local	2007-08-08 20:55:25.278998342 +0200
@@ -34,7 +34,7 @@
 		;;
 *)		(
 			sleep 5
-			ddclient -daemon=0 -syslog -use=if -if=$1 >/dev/null 2>&1
+			ddclient -daemon=0 -syslog -use=if -if=$IP >/dev/null 2>&1
 		) &
 		;;
 esac
