From: Torsten Landschoff <torsten@debian.org>
Date: Tue, 18 Oct 2005 22:54:26 +0200
Subject: From version 3.6.2-5, maxinterval.diff

Forwarded: no
---
 ddclient | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ddclient b/ddclient
index f2acbfc..3b40f57 100755
--- a/ddclient
+++ b/ddclient
@@ -393,7 +393,7 @@ my %variables = (
 	'atime'               => setv(T_NUMBER, 0, 1, 0, 0,                   undef),
 	'status'              => setv(T_ANY,    0, 1, 0, '',                  undef),
 	'min-interval'        => setv(T_DELAY,  0, 0, 1, interval('30s'),     0),
-	'max-interval'        => setv(T_DELAY,  0, 0, 1, interval('25d'),     0),
+	'max-interval'        => setv(T_DELAY,  0, 0, 1, interval('30d'),     0),
 	'min-error-interval'  => setv(T_DELAY,  0, 0, 1, interval('5m'),      0),
 
 	'warned-min-interval'       => setv(T_ANY,    0, 1, 0, 0,             undef),
