Index: ddclient-3.7.0/ddclient
===================================================================
--- ddclient-3.7.0.orig/ddclient	2007-01-20 11:41:28.344052373 +0100
+++ ddclient-3.7.0/ddclient	2007-01-20 11:41:52.034438257 +0100
@@ -1660,7 +1660,7 @@
  
     debug("server = $server");
     debug("opt(fw = ",opt('fw'));
-    debug("glo fw = $globals{'fw'}");
+    debug(defined $globals{fw} ? "glo fw = $globals{'fw'}" : "glo fw unset");
     #if ( $globals{'ssl'} and $server ne $globals{'fw'} ) {
     ## always omit SSL for connections to local router
     if ( $globals{'ssl'} and (caller(1))[3] ne 'main::get_ip' ) {
