Title: From version 3.8.1, smc-barricade-fw-alt.diff
DEP: 3
Forwarded: no
--- a/ddclient
+++ b/ddclient
@@ -88,6 +88,11 @@ my %builtinfw = (
 				  'url'  => '/status.HTM',
 				  'skip' => 'WAN IP',
 			        },
+    'smc-barricade-alt'      => {
+				  'name' => 'SMC Barricade FW (alternate config)',
+				  'url'  => '/status.HTM',
+				  'skip' => 'WAN IP',
+			        },
     'smc-barricade-7401bra'  => {
 				  'name' => 'SMC Barricade 7401BRA FW',
 				  'url'  => '/admin/wan1.htm',
