Index: ddclient-3.7.3/ddclient
===================================================================
--- ddclient-3.7.3.orig/ddclient	2007-08-08 20:55:25.246003376 +0200
+++ ddclient-3.7.3/ddclient	2007-08-08 20:56:34.117496598 +0200
@@ -77,6 +77,11 @@
 				  'url'  => '/status.htm',
 				  'skip' => 'IP Address',
 			        },
+    'smc-barricade-alt'      => {
+				  'name' => 'SMC Barricade FW (alternate config)',
+				  'url'  => '/status.HTM',
+				  'skip' => 'WAN IP',
+			        },
     'smc-barricade-7401bra'  => {
 				  'name' => 'SMC Barricade 7401BRA FW',
 				  'url'  => '/admin/wan1.htm',
