From: Torsten Landschoff <torsten@debian.org>
Date: Fri, 10 Aug 2012 02:10:02 +0200
Subject: From version 3.6.7-1, smc-barricade-fw-alt.diff

Forwarded: no
---
 ddclient | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/ddclient b/ddclient
index 8a1bacc..09c5fbc 100755
--- a/ddclient
+++ b/ddclient
@@ -92,6 +92,11 @@ my %builtinfw = (
 				  'url'  => '/status.HTM',
 				  'skip' => 'WAN IP',
 			        },
+    'smc-barricade-alt'      => {
+				  'name' => 'SMC Barricade FW (alternate config)',
+				  'url'  => '/status.HTM',
+				  'skip' => 'WAN IP',
+			        },
     'smc-barricade-7401bra'  => {
 				  'name' => 'SMC Barricade 7401BRA FW',
 				  'url'  => '/admin/wan1.htm',
